src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -1340,6 +1340,7 @@ public void encodeObject(OutputStream out) {
     }
 
     public static Configuration read(File file) throws IOException {
+        LOGGER.log(Level.INFO, "Reading configuration from {0}", file.getCanonicalPath());
```

Comment:
Perhaps just logging `file` would be good enough, since `getCanonicalPath()` can throw `IOException`.

Reply:
I wanted to resolve any symlinks however on my Linux system getCanonicalPath() does not do that and reports the path without resolving the symlink.

Reply:
Even if it threw an exception while resolving the path, it would be caught in the same block in `contextInitialized()` so the reporting would be the same.